hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestAsyncTableTracing.java

```diff
@@ -452,6 +456,53 @@ public void testScanAll() {
     assertTrace("SCAN");
   }
 
+  @Test
+  public void testScan() throws Throwable {
```

Comment:
Name: testScanTracing?

Reply:
The class has "Tracing" in the name, so I didn't think it necessary.